src/python/pants/backend/codegen/protobuf/lint/buf/lint_rules_integration_test.py

```diff
@@ -60,10 +60,10 @@ def run_buf(
         [BufLintRequest.PartitionRequest(tuple(BufFieldSet.create(tgt) for tgt in targets))],
     )
     results = []
-    for key, subpartition in partition.items():
+    for key, Batch in partition.items():
```

Comment:
```suggestion
    for key, batch in partition.items():
```

Reply:
and everywhere below in this file and in:
- src/python/pants/backend/docker/lint/hadolint/rules_integration_test.py
- src/python/pants/backend/go/lint/golangci_lint/rules_integration_test.py
- src/python/pants/backend/go/lint/vet/rules_integration_test.py
- src/python/pants/backend/helm/goals/lint_test.py
- src/python/pants/backend/openapi/lint/spectral/rules_integration_test.py

(by the way if you're using VS Code (or likely another editor with the feature) there's usually a "preserve case" option for Find-and-Replace :wink:

Reply:
:hurtrealbad: good catch